### PR TITLE
Cross-link to Flutter pubspec extensions

### PR DESCRIPTION
**Pull request by mit-mit**
Add a reference to the Flutter specific extensions to pubspec

Fixes https://github.com/dart-lang/site-www/issues/421

### CODE REVIEW

**Comment by mit-mit** [comment]
@Sfshaza any chance you can review this now that you are likely(?) back from last weeks holiday?

**Comment by lbrawnbrawny** [comment]
good

File: src/tools/pub/pubspec.md [hunk 1]
```diff
@@ -57,8 +57,14 @@ For more information, see
 [Pub Assets and Transformers](/tools/pub/assets-and-transformers) and
 [Configuring the Built-in dart2js Compiler]({{site.webdev}}/tools/pub/dart2js-transformer).
 
-All other fields are ignored. A simple but complete pubspec looks something
-like the following:
+All other fields are ignored. 
+
+**Note:** The pubspec for [Flutter apps](https://flutter.io) supports
+[a few additional fields](https://flutter.io/assets-and-images/) for managing assets.
```
Comment (chalin) [inline src/tools/pub/pubspec.md]: Replace `https://flutter.io` by `{{site.flutter}}`. Also, I'd suggest this wording:

    **Note:** [Flutter app]({{site.flutter}}) pubspecs support [a few additional fields]({{site.flutter}}/assets-and-images) for managing assets.

File: src/tools/pub/pubspec.md [hunk 2]
```diff
@@ -57,8 +57,14 @@ For more information, see
...
 [Configuring the Built-in dart2js Compiler]({{site.webdev}}/tools/pub/dart2js-transformer).
 
-All other fields are ignored. A simple but complete pubspec looks something
-like the following:
+All other fields are ignored. 
+
+**Note:** The pubspec for [Flutter apps](https://flutter.io) supports
+[a few additional fields](https://flutter.io/assets-and-images/) for managing assets. 
+
+## Sample pubspec
```
Comment (chalin) [inline src/tools/pub/pubspec.md]: I'd suggest dropping this heading.

**Comment by chalin** [comment]
@mit-mit - let me know if you'd like for me to make the changes. If you do the updates, I can merge afterwards. Either way is fine.

File: src/tools/pub/pubspec.md (hunk 2 above)
Comment (mit-mit) [reply src/tools/pub/pubspec.md]: Updated the heading. But think we should keep it to make sure it doesn't seem like a sample flutter pubspec given the paragraph just before it.

**Comment by mit-mit** [comment]
@chalin PR updated, please go ahead and merge

**Pull request merged**